src/http/routing.rs

```diff
@@ -1043,11 +1217,41 @@ impl<'a> RouteInfo<'a> {
                 Route::GuildsIdEmojisId(guild_id),
                 Cow::from(Route::guild_emoji(guild_id, emoji_id)),
             ),
+            RouteInfo::DeleteFollowupMessage {
+                application_id,
+                interaction_token,
+                message_id
+            } => (
+                LightMethod::Delete,
+                Route::WebhooksApplicationId(application_id),
+                Cow::from(Route::webhook_followup_message(
+                    application_id,
+                    interaction_token,
+                    message_id
+                )),
+            ),
+            RouteInfo::DeleteGlobalApplicationCommand {
+                application_id,
+                command_id
+            } => (
+                LightMethod::Delete,
+                Route::ApplicationsIdCommandsId(application_id),
+                Cow::from(Route::application_command(application_id, command_id)),
+            ),
             RouteInfo::DeleteGuild { guild_id } => (
                 LightMethod::Delete,
                 Route::GuildsId(guild_id),
                 Cow::from(Route::guild(guild_id)),
             ),
+            RouteInfo::DeleteGuildApplicationCommand {
+                application_id,
+                guild_id,
+                command_id
```

Comment:
```suggestion
                command_id,
```
Missing trailing comma.